Source/DafnyLanguageServer.Test/Various/ShutdownTest.cs

```diff
@@ -71,12 +71,21 @@ public class ShutdownTest {
 
       // Wait for the language server to kill itself by waiting until it closes the output stream.
       await process.StandardOutput.ReadToEndAsync();
-      await Task.Delay(400); // Give the process some time to die
 
-      Assert.ThrowsException<ArgumentException>(() => {
-        var languageServer = Process.GetProcessById(int.Parse(languageServerProcessId));
-        languageServer.Kill();
-      }, "Language server should have killed itself if the parent is gone.");
+      for (int attempts = 0; attempts < 10; attempts++) {
+        // Give the language server time to kill itself
+        await Task.Delay(100);
+
+        try {
+          Process.GetProcessById(int.Parse(languageServerProcessId));
+        } catch {
+          return;
```

Comment:
Shouldn't this be a `continue`?

Reply:
The catch is the happy case, in which the language server has killed itself and the test should pass.